addons/website_forum_doc/controllers/main.py

```diff
@@ -1,22 +1,17 @@
 # -*- coding: utf-8 -*-
 
-
 from openerp import http
 from openerp.http import request
-from openerp.addons.website.models.website import slug
 
 
 class WebsiteDoc(http.Controller):
     @http.route(['/forum/how-to', '/forum/how-to/<model("forum.documentation.toc"):toc>'], type='http', auth="public", website=True)
     def toc(self, toc=None, **kwargs):
-        cr, uid, context, toc_id = request.cr, request.uid, request.context, False
         if toc:
             sections = toc.child_ids
             forum = toc.forum_id
         else:
-            toc_obj = request.registry['forum.documentation.toc']
-            obj_ids = toc_obj.search(cr, uid, [('parent_id', '=', False)], context=context)
-            sections = toc_obj.browse(cr, uid, obj_ids, context=context)
+            sections = request.env['forum.documentation.toc'].search([('parent_id', '=', False)])
             forum = sections and sections[0].forum_id or False
```

Comment:
Search with limit=1return only one section, so it show only one Documentation. 
In value there is pass multi sections.